ehtim/array.py

```diff
@@ -127,22 +170,24 @@ def make_subarray(self, sites):
         """
         all_sites = [t[0] for t in self.tarr]
         mask = np.array([t in sites for t in all_sites])
-        return Array(self.tarr[mask],ephem=self.ephem)
+        return Array(self.tarr[mask], ephem=self.ephem)
 
     def save_txt(self, fname):
         """Save the array data in a text file.
 
            Args:
                fname (str) : path to output array file
         """
-        ehtim.io.save.save_array_txt(self,fname)
+        ehtim.io.save.save_array_txt(self, fname)
         return
```

Comment:
I know this has nothing to do with this PR, but what does this function return? 
```suggestion
       
```
IMO this is best